com.ibm.wala.core.java11/build.gradle

```diff
@@ -4,8 +4,55 @@ plugins {
 
 eclipse.project.natures 'org.eclipse.pde.PluginNature'
 
+java.sourceCompatibility = null
+java.targetCompatibility = null
```

Comment:
It's a bit gross to have to null these out.  Maybe we can move this language version configuration to the top-level `build.gradle` and do it conditionally based on the sub-project name?  Or will that mess up Eclipse / Buildship?

Reply:
The only package that should have a different configuration for these settings is `com.ibm.wala.core.java11`. Is hard-coding that sub-project name in the top-level `build.gradle` less gross that reassigning these settings to `NULL` here?

Reply:
Fair enough.  We do some conditional stuff on project name [here](https://github.com/wala/WALA/blob/58ca8a9640f1b0e07ecd9e2cf775f8fba68201b2/build.gradle#L46-L53), but that's a bit different I guess.  It's your call on how best to do this, but if you decide to go with the `null` assignments maybe add a comment as to why it's needed.

Reply:
> if  you decide to go with the `null` assignments maybe add a comment as to why it's needed.

Done!